docs/java/testing-and-debugging.md

```diff
@@ -110,6 +137,10 @@ public class HelloActivityTest {
 }
 ```
 
+You can find all unit tests for the [Temporal Java samples](https://github.com/temporalio/samples-java) repository [here](https://github.com/temporalio/samples-java/tree/master/src/test/java/io/temporal/samples).
```

Comment:
For SEO, recommend changing link text to something like "repository in [temporal/samples-java]", per [Link text](https://developers.google.com/style/link-text) in the Google dev doc style guide. (Finding suitable link text when linking into a repo is definitely trickier than  when linking to an ordinary page.)